Setting-And-UI Update-->Pre Code Driven UI - PLEASE ACCEPT

dustinwloring1988 opened the pull request:
This is the last time I will change the SCENE file before we start on a code driven UI. The settings file started using ...addListener(....) to help in the future.
Also add the trade test to the main SCENE by putting a button next to path finding

dustinwloring1988 commented:
![12](https://cloud.githubusercontent.com/assets/21135165/17833224/d89d7cc0-66e4-11e6-8122-8c342822f45e.png)

OzieGamma commented on Assets/Scripts/Settings/GameSettings.cs:
```diff
@@ -0,0 +1,13 @@
+﻿public class GameSettings {
```
brace on the next line

OzieGamma commented on Assets/Scripts/UI/SettingHelper.cs:
```diff
@@ -12,25 +12,133 @@ public class SettingHelper : MonoBehaviour {
+		myResolutions = Screen.resolutions;
+
+		//Add our listeners
+		fpsToggle.onValueChanged.AddListener(delegate { OnFPSToggle(); } );
```
`fpsToggle.onValueChanged.AddListener(OnFPSToggle);` is shorter and should also work

OzieGamma commented on Assets/Scripts/UI/SettingHelper.cs:
```diff
@@ -12,25 +12,133 @@ public class SettingHelper : MonoBehaviour {
+		vSyncDropdown.onValueChanged.AddListener(delegate { OnVSyncChange(); } );
+		quilityDropdown.onValueChanged.AddListener(delegate { OnQuilityChange(); } );
+
+		//masterVolume.onValueChanged.AddListener(delegate { OnMasterChange(); } );
```
No commented code please.

OzieGamma commented on Assets/Scripts/UI/SettingHelper.cs:
```diff
@@ -12,25 +12,133 @@ public class SettingHelper : MonoBehaviour {
+	{
+		/*
+		for (int i = 0; i < myResolutions.Length; i++) {
+			resolutionDropdown.AddOptions (myResolutions [i].height.ToString () + "x" + myResolutions [i].width.ToString () + "   " + myResolutions [i].refreshRate.ToString ());
```
No commented code please.

OzieGamma commented:
Thanks for the PR!

What did you change in the scene? Are those changes needed?

dustinwloring1988 commented:
@OzieGamma Yes made changes to the setting panel and added a button to the options next to path finding for the trade test someone made. And removed commented code and the shorter way of adding a listener did not work.....

OzieGamma commented:
@dustinwloring1988 Sorry about that, my bad.

dustinwloring1988 commented:
@OzieGamma No its fine Im not the best programmer and any help right or wrong will help me learn..

dustinwloring1988 commented:
Please add so we can start Code Driven UI

OzieGamma commented:
I don't have Unity on my machine, so I can't try it out but I try to give help with my C# experience. That's how you learn !

dustinwloring1988 commented:
https://github.com/TeamPorcupine/ProjectPorcupine/issues/404  -  code driven UI coming soon......

dustinwloring1988 commented:
Added more code driven UI calls

dustinwloring1988 commented:
made a separate unity scene until it is complete
